enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/AccessLoggingService.java

```diff
@@ -43,6 +44,8 @@
 import java.util.concurrent.LinkedBlockingQueue;
 import java.util.concurrent.TimeUnit;
 
+import static org.wso2.choreo.connect.enforcer.commons.logging.ErrorDetails.errorLog;
+
```

Comment:
Lets try not to use the static import. There are other places as well in the same PR

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2666/commits/51303937f8b421660770bc244d87f82ab1951668